ospf6d/ospf6_lsdb.c

```diff
@@ -92,6 +92,44 @@ static void _lsdb_count_assert(struct ospf6_lsdb *lsdb)
 #define ospf6_lsdb_count_assert(t) ((void) 0)
 #endif /*DEBUG*/
 
+static void ospf6_lsdb_stats_add(struct ospf6_lsa *lsa, struct ospf6_lsdb *lsdb,
+				 int count)
+{
+	uint32_t *num = NULL;
+
+	switch (ntohs(lsa->header->type)) {
+	case OSPF6_LSTYPE_ROUTER:
+		num = &lsdb->stats.num_type1;
+		break;
+	case OSPF6_LSTYPE_NETWORK:
+		num = &lsdb->stats.num_type2;
+		break;
+	case OSPF6_LSTYPE_INTER_PREFIX:
+		num = &lsdb->stats.num_type3;
+		break;
+	case OSPF6_LSTYPE_INTER_ROUTER:
+		num = &lsdb->stats.num_type4;
+		break;
+	case OSPF6_LSTYPE_AS_EXTERNAL:
+		num = &lsdb->stats.num_type5;
+		break;
+	case OSPF6_LSTYPE_TYPE_7:
+		num = &lsdb->stats.num_type7;
+		break;
+	case OSPF6_LSTYPE_LINK:
+		num = &lsdb->stats.num_type8;
+		break;
+	case OSPF6_LSTYPE_INTRA_PREFIX:
+		num = &lsdb->stats.num_type9;
+		break;
```

Comment:
Please add a counter for Grace lsa (OSPF6_LSTYPE_GRACE_LSA)

Reply:
Done.